src/rules/converters/variable-name.ts

```diff
@@ -0,0 +1,121 @@
+import { RuleConverter } from "../converter";
+
+export const convertVariableName: RuleConverter = tslintRule => {
+    const getCamelCaseRuleOptions = () => {
+        const camelCaseOptionNotice: string[] = [];
+        if (tslintRule.ruleArguments.includes("check-format")) {
+            if (
+                !tslintRule.ruleArguments.includes("allow-leading-underscore") &&
+                !tslintRule.ruleArguments.includes("allow-trailing-underscore")
+            ) {
+                camelCaseOptionNotice.push(
+                    "Leading and trailing underscores (_) in variable names will now be ignored.",
+                );
+            } else if (
+                tslintRule.ruleArguments.includes("allow-leading-underscore") &&
+                !tslintRule.ruleArguments.includes("allow-trailing-underscore")
+            ) {
+                camelCaseOptionNotice.push(
+                    "Leading undescores in variable names will now be ignored.",
+                );
+            } else if (
+                !tslintRule.ruleArguments.includes("allow-leading-underscore") &&
+                tslintRule.ruleArguments.includes("allow-trailing-underscore")
+            ) {
+                camelCaseOptionNotice.push(
+                    "Trailing undescores in variable names will now be ignored.",
+                );
+            }
+        } else {
+            camelCaseOptionNotice.push(
+                "Leading and trailing underscores (_) in variable names will now be ignored.",
+            );
+        }
```

Comment:
I'm finding this section a little difficult to read... Would you mind changing it to something like:

```ts
const allowedLeadingUnderscore = tslintRule.ruleArguments.includes("allow-leading-underscore");
const allowedTrailingUnderscore = tslintRule.tslintRuleArguments.includes("allow-trailing-underscore");

if (allowedLeadingUnderscore) {
    if (!allowedTrailingUnderscore) {
        camelCaseOptionNotice.push("...");
    }
} else if (allowedTrailingUnderscore) {
     // ...
} else {
     // ...
}
```

_(and similar changes for later functions, if they end up being smaller this way?)_

Reply:
YES! Can do it :) I do get confused also with those nested if-else statements, was thinking on using switch case instead.